# HG changeset patch
# User David Bateman <dbateman@free.fr>
# Date 1297618230 -3600
#      Sun Feb 13 18:30:30 2011 +0100
# Node ID 278902427e875ab3cae438262fc8e1aed16d5773
# Parent  b932ae4381da1a8d233cfcb04627fb10c676ba21
Add changelog to change acccidentally pushed in afb65a7bc065

diff --git a/liboctave/ChangeLog b/liboctave/ChangeLog
--- a/liboctave/ChangeLog
+++ b/liboctave/ChangeLog
@@ -1,8 +1,14 @@
+2011-02-13  David Bateman  <dbateman@free.fr>
+
+	* Sparse-perm-op-defs.h (template <typename SM>
+	SM octinternal_do_mul_colpm_sm (const octave_idx_type *,
+	const SM&)): Ensure that the row indices are sorted.
+
 2011-02-08  John W. Eaton  <jwe@octave.org>
 
 	* oct-alloc.h: Include <cstddef>.
 	From Orion Poplawski <orion@cora.nwra.com>.
 
 2011-02-06  John W. Eaton  <jwe@octave.org>
 
 	* PermMatrix.cc (operator*): Fix mixed row/column permutation
